src/Core/src/Platform/iOS/MauiTextView.cs

```diff
@@ -112,6 +112,17 @@ public override NSAttributedString AttributedText
 				}
 			}
 		}
+		
+		public override NSAttributedString? AccessibilityAttributedValue
+		{
+			get
+			{
+				return string.IsNullOrWhiteSpace(Text)
```

Comment:
Hold off on merging this one

Need to validate a little bit here with how PH scenarios work. 

I think setting the value of this to the PH might be confusing behavior because it's not the actual value of the field

Reply:
I need to do a little bit more research, but, I think the fix here will be to propagate the PlaceHolder value to AccessibilityLabel

When I test different platform apps (like the contacts app) it looks like they always set the "AccessibilityLabel" to the same value of the PH

Though we might not want to do this for users, because some users might want to use a separate field for the PH and not announce the PH value at all....

Hmmmm